Including glog/logging.h triggers -Wunused-parameter on g++

The following script demonstrates the issue:

```
#!/bin/bash
# install glog to glog_prefix
git clone https://github.com/google/glog.git
mkdir -p glog_prefix
cd glog
mkdir build
cd build
cmake3 -DCMAKE_INSTALL_PREFIX=../../glog_prefix \
       -DCMAKE_BUILD_TYPE=Release \
       -DWITH_GFLAGS=OFF \
       ..
make -j$(nproc --ignore=1)
make install
cd ../..

# build a cpp file that includes glog
cat <<EOF > test.cpp
#include <glog/logging.h>
int main(int, char**) { return 0; }
EOF
g++ -o test -Wall -Wextra -Iglog_prefix/include -Lglog_prefix/lib64 -lglog test.cpp
```

Expected output: no compiler warning
Actual output: 
```
In file included from test.cpp:1:0:
glog_prefix/include/glog/logging.h:1434:16: warning: unused parameter 'usecs' [-Wunused-parameter]
   virtual void send(LogSeverity severity, const char* full_filename,
                ^
```

Operating system: CentOS 7.6.1810
gcc version: 4.8.5 (also tested with gcc 8.2.0)

--- a/src/glog/logging.h.in
+++ b/src/glog/logging.h.in
@@ -1438,7 +1438,7 @@ class GOOGLE_GLOG_DLL_DECL LogSink {
   virtual void send(LogSeverity severity, const char* full_filename,
                     const char* base_filename, int line,
                     const struct ::tm* tm_time,
-                    const char* message, size_t message_len, int32 usecs) {
+                    const char* message, size_t message_len, int32 /*usecs*/) {
     send(severity, full_filename, base_filename, line,
          tm_time, message, message_len);
   }

